style.css

```diff
@@ -446,3 +480,136 @@ nav {
   text-align: center;
   height: 200px;
 }
+
+/* |||----------------------------desktop-support---------- */
+@media only screen and (max-width: 992px) {
+  header nav.desktop {
+    display: none;
+  }
+}
+
+@media only screen and (min-width: 992px) {
+  header nav.mobile {
+    display: none;
+  }
+
+  header nav.desktop {
+    display: block;
+  }
+
+  .works {
+    grid-template-columns: repeat(3, 1fr);
+    grid-template-rows: repeat(4, auto);
+    margin: 142px;
+  }
+
+  .works .title,
+  .works > img {
+    grid-column: 1 / -1;
+  }
+
+  .about-me {
+    margin: 142px 0;
+    display: grid;
+    grid-template-rows: repeat(2, auto);
+    grid-template-columns: 142px repeat(3, 1fr) 142px;
+    gap: 24px;
+    min-height: 90vh;
+    justify-content: space-around;
+  }
+
+  .about-me .text-section {
+    grid-column: 1 / -1;
+    background-image:
+      url(./resources/about-me-desktop-bg-right.svg),
+      url(./resources/about-me-desktop-bg-left.svg);
+    background-repeat: no-repeat, no-repeat;
+    background-size: auto 40%, auto 20%;
+    background-position: top right, bottom left;
+    margin: 0;
+  }
+
+  .about-me .about-me-card {
+    margin: 0;
+  }
+
+  .about-me .about-me-card:nth-child(2) {
+    margin-left: 142px;
+    grid-column: 1 / 3;
+  }
```

Comment:
- From the "About me" section, please kindly vertically center those containers, like in the Figma template.

Your Implementation | Figma Design
:---:|:---:
![image](https://user-images.githubusercontent.com/57421823/149323266-5b078458-70a7-4c2d-8ac5-68d0af529fd8.png) | ![image](https://user-images.githubusercontent.com/57421823/149323375-3d8ae789-778b-4f07-8ca6-d4c19f695cdf.png)

That way, the project will look a lot more like the template! 💪

Reply:
Good catch. I didn't notice this. :thumbsup: :thumbsup: